src/core/display/DisplayObject.js

```diff
@@ -431,6 +431,25 @@ DisplayObject.prototype.generateTexture = function (renderer, scaleMode, resolut
 };
 
 /**
+ * Set the parent Container of this DisplayObject
+ *
+ * @param container {Container} The Container to add this DisplayObject to
+ * @return {DisplayObject} This DisplayObject
+ */
+DisplayObject.prototype.setParent = function (container)
+{
+    if (!container || !container.addChild)
+    {
+        throw new Error('setParent: Argument must be a Container');
+    }
+    else
+    {
+        container.addChild(this);
+        return this;
```

Comment:
Nice catch. Is there any reason `Container.addChild` returns the child instead of the `Container`? The `Container` returning itself would likely be more useful (for chaining). If you're amenable, I'll submit a PR for changing the return value of `Container.addChild`, otherwise I'll go ahead and make the requested changes to `DisplayObject.setParent`.